### ISSUE TITLE
Are the components accessible? 

### ISSUE THREAD

**Issue by nikoma** [SamProf/MatBlazor#364]
Do the components work with screen readers for the visual impaired?

**Comment by enkodellc** [comment]
@nikoma I agree that it should be something addressed / answered. I have not dealt with this requirement before so I could not tell you what is required to meet them. I recommend reviewing  material design documentation:  https://material.io/design/usability/accessibility.html#understanding-accessibility
 as well as tools like this to answer your question:  https://www.sitepoint.com/site-accessibility-tools/ 

Being an open source project these advanced features may require contributions for features like this.

**Comment by Christian-Oleson** [comment]
@nikoma ,
We are always looking for contributors.

**Comment by KasperPi** [comment]
Greetings,
I am new to Blazor and this is my first Github comment. I signed up specifically to be able to ask this question:

Is there a way to access all of the Mat components via the keyboard?
Example: using only the keyboard to get to a tab within MatTabGroup or tab into a MatAccordion and open close the panels using the spacebar

Any MatBlazor parts that cannot be reached via the keyboard makes a site inaccessible. If it's not possible to accomplish this, MatBlazor is relegated to never be usable for government, bank, or corporate site. This applies to most countries that have accessibility requirements/laws (eg US/Canada etc).

I have started using MatBlazor but I cannot seem to reach MatTabGroups with just the keyboard. As it stand now, I will have to remove all of the MatBlazor stuff because I cannot accomplish this. Perhaps it's just because I am new that I cannot figure out how to do this.
